src/components/Form/Form.test.tsx

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import "@testing-library/jest-dom/extend-expect";
+import {
+  Form,
+  Input,
+  Button,
+  useBasisForm,
+} from "../..";
+
+import { render, screen, userEvent, act, fireEvent } from "../../utils/test";
+import { SubmitHandler } from "react-hook-form";
+
+
+interface SimpleFormValues {
+  testInput: string;
+}
+
+interface SimpleProps {
+  onSubmit: SubmitHandler<SimpleFormValues>;
+  validate?: any;
+}
+
+const SimpleForm = ({ onSubmit, validate }: SimpleProps) => {
+  const { methods, Field } = useBasisForm<SimpleFormValues>();
+
+  return (
+    <Form methods={methods} onSubmit={onSubmit}>
+      <Field label="Test" name="testInput" as={Input} validate={validate} />
+      <Button type="submit">Submit</Button>
+    </Form>
+  )
+}
+
+describe('Form', () => {
+  it('should render the form', () => {
+    render(<SimpleForm onSubmit={() => { }} />)
```

Comment:
Can we make `onSubmit` optional?

Reply:
Then why do we have a Form? just don't use a form at that point if you don't care about the on submit